examples/tv-app/android/java/ContentAppCommandDelegate.cpp

```diff
@@ -81,11 +81,13 @@ void ContentAppCommandDelegate::InvokeCommand(CommandHandlerInterface::HandlerCo
             env->ExceptionDescribe();
```

Comment:
https://docs.oracle.com/en/java/javase/16/docs/specs/jni/functions.html#exceptiondescribe seems to indicate that "The pending exception is cleared as a side-effect of calling this function" ... do we need to also call ExceptionClear ?